custom_components/tuya_local/devices/solar_inverter_gtb_series.yaml

```diff
@@ -0,0 +1,150 @@
+name: Solar Inverter GTB Series
+products:
+  - id: czu7ynosrkxgbnsb
+    name: GTB Series
+primary_entity:
+  entity: sensor
+  class: energy
+  dps:
+    - id: 2
+      type: integer
+      name: sensor
+      unit: kWh
+      class: total_increasing
+      mapping:
+        - scale: 100
+    - id: 15
+      name: model
+      type: string
+    - id: 16
+      name: inverter_id
+      type: string
+secondary_entities:
+  - entity: sensor
+    class: power
+    category: diagnostic
+    dps:
+      - id: 10
+        type: integer
+        name: sensor
+        unit: W
+        force: true
```

Comment:
Normally `force: true` should only be needed when the device does not send this regularly. In those cases, `optional: true` would also be needed to ensure the config can be matched reliably.